tests_integration/__tests__/__snapshots__/config-resolution.js.snap

```diff
@@ -58,7 +58,7 @@ exports[`accepts configuration from --config 1`] = `
 "/* eslint-disable */
 
 function f() {
-  console.log(\\"should have tab width 8\\")
+        console.log(\\"should have tab width 8\\");
```

Comment:
I'm not sure how could this happen?! I'm just refactoring...

Reply:
Looks like the `--config` argument isn't being respected.

Reply:
Ah I just realized. cosmiconfig reads `process.argv` itself

https://github.com/davidtheclark/cosmiconfig/blob/6eb3cc85d3a435e6b9e252b341e601960312f410/index.js#L9